BrickController2/BrickController2/Resources/TranslationResources.resx

```diff
@@ -609,4 +609,7 @@
   <data name="Add" xml:space="preserve">
     <value>Add</value>
   </data>
+  <data name="AddControllerEventWithControllerId" xml:space="preserve">
+    <value>Add a new controller event with ControllerId</value>
```

Comment:
It seems to me to programish 😄 
May be something like:
- Add a new controller event for given game controller
- Add a new controller event for specific game controller

Reply:
I changed the tooltip to "Add a new controller event for specific game controller"
https://github.com/vicocz/brickcontroller2/pull/75/commits/f6714ade0fa839fdc12bfc2f504eae4ee52c8784